com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Propagating-NaNs.md

```diff
@@ -0,0 +1,52 @@
+
+## Propagating NaNs/Infs
+
+Any operation that has a NaN or an Inf as an operand also produces a NaN/Inf as a result. This is important in the High Definition Render Pipeline (HDRP), because any filtering or blur HDRP performs that involves NaN/Inf values spreads the invalid values further. 
+
+A commonly reported issue in HDRP is that [bloom](Post-Processing-Bloom.md) generates a black screen. The bloom effect does not produce NaNs/Infs itself, but it spreads a NaN/Inf generated elsewhere across the screen. This happens because to calculate bloom, HDRP downsamples/filters the scene color and then upsamples it back to the desired resolution. If there is a single NaN/Inf on the screen, the downsample process spreads the invalid value until it covers the whole texture, and the subsequent upsampling therefore contains only NaN values, which leads to the full screen being black. 
+
+For example, see a NaN caused by a material issue that spreads to the whole scene when HDRP calculates bloom:
+
+![](Images/Post-processingPropagatingNaNsExample1.png)
+
+![](Images/Post-processingPropagatingNaNsExample2.png)
+
+![](Images/Post-processingPropagatingNaNsExample3.png)
+
+![](Images/Post-processingPropagatingNaNsExample4.png)
+
+A similar issue occurs when HDRP generates color pyramids for use by features such as [screen-space reflection](Override-Screen-Space-Reflection.md), [screen-space refraction](Override-Screen-Space-Refraction.md), and distortion. 
+
+If you disable bloom and the screen stops being black, the cause of the black screen is likely because a single NaN/Inf pixel is present, but not really visible, and bloom propagated it across the whole screen. It is **not** because bloom created the invalid values.
+
+## Fixing NaNs and Infs
+
+The best way to stop bloom or other HDRP features from propagating NaN/Inf values is to fix the source of the NaN/Inf values. For information on how to do this, see [finding NaNs and Infs](#finding-nans-and-infs).
+
+If you are unable to fix the source of the NaN/Inf values, [HDRP Cameras](HDRP-Camera.md) include a feature which replaces NaN and Inf values with a black pixel. This stops effects like bloom from propagating NaN/Inf values, but is a fairly resource intensive process. To enable this feature, select a Camera and, in the Inspector, enable the **Stop NaNs** checkbox. Note that you should only enable this feature if you are unable to fix the root cause of the NaN/Inf values.
+
+### Finding NaNs and Infs
```

Comment:
We also have a debug mode just for this in the Debug Window. It should display pixels containing NaNs with a recognizable color.
It's under Rendering -> Fullscreen Debug Mode -> NaN Tracker.
Might be better to mention this first as it's easier for people to use to track if there is actually NaNs on the screen (but then render doc will be more useful to track the actual draw call containing the issue).